src/components/Homepage/events.js

```diff
@@ -81,55 +78,50 @@ const Events = ({ events }) => (
     </Col>
 
     <Col width={[1, 1, 1, 1, 8 / 12, 8 / 12, 8 / 12]}>
-      {events
-        .filter(n => n.node.homepageFeatured)
-        .map(({ node }) => (
-          <FeaturedEvent key={node.id} color={node.color}>
-            <EventWrapper>
-              <BodyPrimary muted reverse noPadding>
-                Featured
-              </BodyPrimary>
-              <CardTitle reverse noPadding biggest>
-                {node.eventTitle}
-              </CardTitle>
-              <Padding top={0.5}>
-                <FixedWidthBodyPrimary muted reverse>
-                  {node.blurb.blurb}
-                </FixedWidthBodyPrimary>
-              </Padding>
-              <StyledLink
-                reverse
-                href={node.linkToEvent}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Visit the website
-              </StyledLink>
-            </EventWrapper>
-            <Image image={node.posterImage} />
-          </FeaturedEvent>
-        ))}
+      {filterConferences(events).map(conf => (
+        <FeaturedEvent key={conf.id} color={conf.color}>
+          <EventWrapper>
+            <BodyPrimary muted reverse noPadding>
+              Featured
+            </BodyPrimary>
+            <CardTitle reverse noPadding biggest>
+              {conf.eventTitle}
+            </CardTitle>
+            <Padding top={0.5}>
+              <FixedWidthBodyPrimary muted reverse>
+                {conf.blurb.blurb}
+              </FixedWidthBodyPrimary>
+            </Padding>
+            <StyledLink
```

Comment:
Same here, I **think** you can use the `ExternalAnchor` with the `as={StyledLink}` prop

Reply:
Didin't work :/

I added an external prop to the styled link